ArduCopter/ReleaseNotes.txt

```diff
@@ -116,6 +116,16 @@ Changes from 4.4.4
 8) Parameters renamed
     - COMPASS_TYPEMASK renamed to COMPASS_DISBLMSK
 9) ROS2 / DDS support
+    - Added support for EProsima MicroXRCEDDS as a transport in SITL and hardware
+    - Added sensor data topic support such as NavSatStatus and BatteryState
+    - Added velocity control support in Copter
+    - Added a new AP_ExternalControl library for generic control support in SI units
+    - Added support for building ArduPilot with the colcon build system
```

Comment:
Also:
- Added ROS 2 launch scripts for SITL, MAVProxy and micro-ROS agent
- Add pytests for DDS client and ROS 2 launch scripts and integrate into CI

Reply:
Added! Thanks!